demos/browser/server.js

```diff
@@ -24,13 +24,13 @@ const indexPage = fs.readFileSync(indexPagePath);
 
 const currentRegion = process.env.REGION || 'us-east-1';
 
-const chimeSDKMediaPipelines = new ChimeSDKMediaPipelines({ region: 'us-east-1' });
-chimeSDKMediaPipelines.endpoint = process.env.CHIME_SDK_MEDIA_PIPELINES_ENDPOINT || "https://media-pipelines-chime.us-east-1.amazonaws.com"
+const chimeSDKMediaPipelines = new ChimeSDKMediaPipelines({
+  region: 'us-east-1',
```

Comment:
Should we use `currentRegion` her as well?

Reply:
I think SDK media pipeline is currently hard-coded to us-east-1 by default. Thought about fixing it but decided not to as it is not related to this PR which is just to fix endpoint syntax for the SDK client.

Reply:
Got it, thanks!